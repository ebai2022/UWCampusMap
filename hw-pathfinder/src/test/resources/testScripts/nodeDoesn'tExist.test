# Testing the node not being in the graph

CreateGraph g

AddNode g n1
AddNode g n2

# Testing if the first node doesn't exist
FindPath g n3 n1

# Testing if the second node doesn't exist
FindPath g n1 n3

# Testing if both nodes don't exist
FindPath g n3 n4