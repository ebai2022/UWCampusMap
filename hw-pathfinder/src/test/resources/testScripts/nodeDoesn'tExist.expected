# Testing the node not being in the graph

created graph g

added node n1 to g
added node n2 to g

# Testing if the first node doesn't exist
unknown: n3

# Testing if the second node doesn't exist
unknown: n3

# Testing if both nodes don't exist
unknown: n3
unknown: n4